src/appshell/qml/Preferences/internal/ProgrammeStartSection.qml

```diff
@@ -51,12 +51,14 @@ BaseSection {
         width: parent.width
 
         delegate: Row {
+            width: parent.width
             spacing: root.columnSpacing
 
             RoundedRadioButton {
                 anchors.verticalCenter: parent.verticalCenter
 
-                width: root.columnWidth
+                width: filePicker.visible ? Math.max(implicitWidth, root.columnWidth)
```

Comment:
I would like to hear the opinion of the designers on this issue.

In your solution, the problem is that the first column on this page may have a different width than first columns from other pages. May look ugly.

Reply:
In principle, the first column _does_ have the same width as on other pages; only in the emergency case that the text in front of the filePicker box needs more space, we give it that space. It is not necessary in English, but we must be prepared for languages where it _is_ necessary. Also, since this page has rather different controls than other pages, I think it won't be a massive problem if this particular page doesn't exactly follow the same column layout in a few languages.